tsnet/tsnet.go

```diff
@@ -822,7 +822,7 @@ func (s *Server) ListenTLS(network, addr string) (net.Listener, error) {
 		return nil, err
 	}
 	if len(st.CertDomains) == 0 {
-		return nil, errors.New("tsnet: you must enable HTTPS in the admin panel to proceed")
+		return nil, errors.New("tsnet: you must enable HTTPS in the admin panel to proceed. See https://tailscale.com/kb/1153/enabling-https/")
```

Comment:
You may want to set up a tailscale.com/s/... shortlink, so we can point the link elsewhere later if we refactor the KB.

Reply:
I agree, I am going to send a clean up PR as we have many KBs linked directly from the CLI.

Reply:
(this one is already linked in the CLI which is why I used it, working on the relevant PRs now)

Reply:
https://github.com/tailscale/tailscale/pull/7549